CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
-  To migrate you'll need to replace any reference to the `grid-width` mixin with `govuk-grid-width`
+If you are not using the Nunjucks macros, add a `govuk-` prefix to `data-module` attribute values. For example:
 
-  ([PR #1342](https://github.com/alphagov/govuk-frontend/pull/1342))
+```html
+<div class="govuk-accordion" data-module="govuk-accordion">
+...
+</div>
+```
 
-- Remove the deprecated `$class` parameter from the `@govuk-grid-column` mixin
+The [button](https://design-system.service.gov.uk/components/button/) and [details](https://design-system.service.gov.uk/components/details/) components now also use the `data-module` attribute for initialisation. If you are not using Nunjucks macros, add:
 
-  To migrate, you'll need to update any places in your code where you use the
-  `@govuk-grid-column` mixin.
+- `data-module="govuk-button"` to each `<button>` HTML tag
+- `data-module="govuk-details"` to each `<details>` HTML tag
 
-  If you are currently passing a class name (`$class: 'foo'`) you should remove
-  the `$class` argument and wrap your call to the mixin in the class name
-  instead:
+If you're using your own JavaScript code to initialise components, add a `govuk-` prefix to any selectors that find components.
 
-  ```
-  .foo {
-    @include govuk-grid-column(...)
-  }
-  ```
+[Pull request #1443: Ensure GOV.UK Frontend component selectors cannot conflict when initialised](https://github.com/alphagov/govuk-frontend/pull/1443)
 
-  If you are currently passing `$class: false` then you can just remove the
-  `$class` argument, as this is now the default behaviour.
+##### Update CSS class names
 
-  ([PR #1376](https://github.com/alphagov/govuk-frontend/pull/1376))
+You do not need to do anything if you're using Nunjucks.
 
-- Ensure GOV.UK Frontend component selectors cannot conflict when initialised
+If you're using HTML or custom JavaScript, change:
 
-  Components which have JavaScript functionality include a data attribute called `[data-module]`,
-  we have added the `govuk-` prefix as a namespace.
+- `js-character-count` to `govuk-js-character-count`
+- `js-header-toggle` in the GOV.UK Frontend header component to `govuk-js-header-toggle`
 
-  This ensures the component's name does not conflict with other service level components
-  with the same name, which is consistent with our CSS class name convention.
+[Pull request #1444: Renames `js-` css prefix to `govuk-js-`](https://github.com/alphagov/govuk-frontend/pull/1444)
 
-  If you're using Nunjucks macros, and are [using the `initAll` function](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) you will not be affected.
+#### If you’re using Nunjucks
 
-  If you are using HTML, you will need to add the `govuk-` prefix to any `[data-module]` attributes.
+1. Change your `nunjucks.configure` property so it only includes the  `node_modules/govuk-frontend/` directory:
```

Comment:
I guess they might still want to import the entire `node_modules` for some other purpose 🤷‍♀ 

The meaning of "only" is a bit opaque here. The array could contain other values.

Reply:
+1 - I think it's very likely that the user will be using nunjucks partials / layouts from elsewhere in their own application.

Reply:
Thanks both - so would it be more like this: "In your `nunjucks.configure` property, change `/frontend/` to `/govuk-frontend/`"?